### DISCUSSION TITLE
Was wondering if you can help me with my dissertation project ?

### DISCUSSION THREAD

**Discussion by samirbenchouk** [community/community#62490] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My dissertation is about Predicting Stock prices. In my case I am using an LSTM model and a various technical indicators. the problem is that now a normal regression model is outperforming my LSTM model. So I thought it might be that my Hyperparameters are not optimal. The dataset I'm using has the following features and goes back to 2008 recording daily Close prices until today: ' Open High Low Close Volume BOLL_Middle BOLL_Upper BOLL_Lower MACD CCI ATR EMA_20 MA5 MA10 MTM6 MTM12 ROC SMI WVAD USDX '
[Copie de Untitled3.ipynb.zip](https://github.com/community/community/files/12241447/Copie.de.Untitled3.ipynb.zip)

is it possible to just get your opinion on my work so far since you are already familiar with this subject

**Comment by queenofcorgis** [comment]
Hi @samirbenchouk,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Posting zip files for members to download will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!
